tests/run-make/output-with-hyphens/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// Rust files with hyphens in their filename should
+// not result in compiled libraries keeping that hyphen -
+// it should become an underscore. Only bin executables
+// should keep the hyphen. This test ensures that this rule
+// remains enforced.
+// See https://github.com/rust-lang/rust/pull/23786
+
+//@ ignore-cross-compile
+
+use run_make_support::rustc;
+use std::path::Path;
+
+fn main() {
+    rustc().input("foo-bar.rs").crate_type("bin").run();
+    Path::new(bin_name("foo-bar")).exists();
```

Comment:
There is now also a `path` shortcut function in `run-make-support` :)

Reply:
Neat! I'll use that.

Reply:
Actually, I don't see it. In which file of `run_make_support` is it?

Reply:
https://github.com/rust-lang/rust/blob/0c960618b56f662d933e8b864cd9632a99174e87/src/tools/run-make-support/src/lib.rs#L86

It was added earlier today :laughing: It is difficult to navigate the multiverse of run-make tests.